src/utilities.ts

```diff
@@ -0,0 +1,3 @@
+export const defaultToTrue = (value: boolean | undefined = true): boolean => {
+    return value;
```

Comment:
If you pass in `null` this would return `null` which is a Falsy value, so might want to safe guard against that too?

Reply:
Good idea!